crypto/nrf_cc310_mbedcrypto/include/mbedtls/ccm_alt.h

```diff
@@ -1,54 +1,27 @@
-*                                                                                     *
-* ARM Object Code and Header Files License, v1.0 Redistribution.                      *
-*                                                                                     *
-* Redistribution and use of object code, header files, and documentation, without     *
-* modification, are permitted provided that the following conditions are met:         *
-*                                                                                     *
-* 1) Redistributions must reproduce the above copyright notice and the                *
-*    following disclaimer in the documentation and/or other materials                 *
-*    provided with the distribution.                                                  *
-*                                                                                     *
-* 2) Unless to the extent explicitly permitted by law, no reverse                     *
-*    engineering, decompilation, or disassembly of is permitted.                      *
-*                                                                                     *
-* 3) Redistribution and use is permitted solely for the purpose of                    *
-*    developing or executing applications that are targeted for use                   *
-*    on an ARM-based product.                                                         *
-*                                                                                     *
-* DISCLAIMER. THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND                  *
-* CONTRIBUTORS "AS IS." ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT             *
-* NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY, NON-INFRINGEMENT,        *
-* AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE          *
-* COPYRIGHT HOLDERS OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,   *
-* SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED            *
-* TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR              *
-* PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF              *
-* LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING                *
-* NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS                  *
-* SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.                        *
-**************************************************************************************/
+/*
+ * Copyright (c) 2001-2019, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause OR Arm’s non-OSI source license
+ */
 
 #ifndef MBEDTLS_CCM_ALT_H
 #define MBEDTLS_CCM_ALT_H
 
-#include <stddef.h>
-#include <stdint.h>
-
 #if defined(MBEDTLS_CONFIG_FILE)
 #include MBEDTLS_CONFIG_FILE
 #endif
 
+#include <stddef.h>
+#include <stdint.h>
+#include "mbedtls/cipher.h"
```

Comment:
Use a < include (unless cipher.h is at mbetls/cipher.h relative from ccm_alt.h)

Reply:
Difficult thing to do, as it breaks with the scheme in mbed TLS upstream. All includes is done in the form "mbedtls/xxxx.h", there... These are files that will end up copied into a generated folder (if in use), but these are also the same that is used internally to resolve include folders when building the internal libraries (without mbedtls include as -isystem)